--- config.def.h.orig	2025-06-03 21:52:37.068841100 -0600
+++ config.def.h.everforest	2025-06-03 21:54:04.976565644 -0600
@@ -96,32 +96,24 @@
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	[0] = "#2f383e", // black
+	[1] = "#e67e80", // red
+	[2] = "#a7c080", // green
+	[3] = "#dbbc7f", // yellow
+	[4] = "#7fbbb3", // blue
+	[5] = "#d699b6", // magenta
+	[6] = "#83c092", // cyan
+	[7] = "#d3c6aa", // white
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+	[8]  = "#4b565c", // bright black
+	[9]  = "#e67e80", // bright red
+	[10] = "#a7c080", // bright green
+	[11] = "#dbbc7f", // bright yellow
+	[12] = "#7fbbb3", // bright blue
+	[13] = "#d699b6", // bright magenta
+	[14] = "#83c092", // bright cyan
+	[15] = "#d3c6aa", // bright white
 };
 
 
@@ -129,9 +121,9 @@
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
-unsigned int defaultcs = 256;
+unsigned int defaultfg = 15;
+unsigned int defaultbg = 0;
+unsigned int defaultcs = 15;
 static unsigned int defaultrcs = 257;
 
 /*
